.buildbot.sh

```diff
@@ -108,7 +126,7 @@ rm temp.bc
 rm lua.bc
 
 cd out-lua
-	make CC=$CC CFLAGS="-DLUA_USE_READLINE_DL -D__LP64__=1 $CONFIG_FLAGS" LDFLAGS="$CONFIG_FLAGS" LLVM_CONFIG=$CHERI/bin/llvm-config
+	make CC=$CC CFLAGS="-DLUA_USE_READLINE_DL -D__LP64__=1 $CONFIG_FLAGS" LDFLAGS="$CONFIG_FLAGS" LLVM_CONFIG=$CHERI/bin/llvm-config 
```

Comment:
Extraneous trailing space.

Reply:
Fix in [6ba25b4](https://github.com/capablevms/llvm-function-split/pull/29/commits/6ba25b47556fdd516172a09f1c0bfdb8f02ce37f).